### PR TITLE
gui: Fix log-tailing behavior when scrolled to top (fixes #7267)

### LINKED ISSUE #7267: GUI log tailing is broken if closed when scrolled up
The log window pauses the live display ("tailing") of the log when scrolled up. If the log window is closed in this state and then reopened, it's empty and says "scroll down to continue", but there is no log data so no scrolling can happen.

Screen Shot 2021-01-12 at 08 38 19

### PR DESCRIPTION
**Pull request by Choongkyu**
### Purpose

Fixes reported issue #7267

### Testing

1) Open logging modal
2) Scroll to top to stop log-tailing
3) Close modal
4) Reopen modal and see log-tailing resumes and displays log instead of blank textarea as shown in issue repoort

### CODE REVIEW

**Review by wweich — CHANGES_REQUESTED** [review]
Instead of creating a new function in `logging`, you should just put `$scope.logging.paused = false;` inside the hidden event handler. Then there is no need to change the markup.

**Comment by calmh** [comment]
Or set paused to false at the top of show()?

File: lib/fs/casefs.go [hunk 1]
```diff
@@ -84,11 +84,23 @@ func (r *caseFilesystemRegistry) get(fs Filesystem) Filesystem {
...
+		toProcess := make([]*caseFilesystem, 0, len(r.fss))
 		for _, caseFs := range r.fss {
-			caseFs.dropCache()
+			toProcess = append(toProcess, caseFs)
 		}
 		r.mut.RUnlock()
+
+		for _, caseFs := range toProcess {
+			caseFs.dropCache()
+		}
```
Comment (imsodin) [inline lib/fs/casefs.go:103]: Unrelated stuff sneaked in.

**Review by imsodin — CHANGES_REQUESTED** [review]

File: lib/fs/casefs.go (hunk 1 above)
Comment (Choongkyu) [reply lib/fs/casefs.go]: thanks for catching that! fixed.

**Review by calmh — APPROVED** [review]

**Review by imsodin — APPROVED** [review]

**Pull request merged**